repos/spack_repo/builtin/packages/intel_mlc/package.py

```diff
@@ -0,0 +1,102 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+import glob
+import os
+
+from spack.package import *
+from spack_repo.builtin.build_systems.generic import Package
+
+
+class IntelMlc(Package):
+    """Intel® Memory Latency Checker (Intel® MLC) is a tool used to measure
+    memory latencies and b/w, and how they change with increasing load on the
+    system. It also provides several options for more fine-grained
+    investigation where b/w and latencies from a specific set of cores to
+    caches or memory can be measured as well."""
+
+    homepage = "https://www.intel.com/content/www/us/en/developer/articles/tool/intelr-memory-latency-checker.html"
+
+    version(
+        "3.11b",
+        url="https://downloadmirror.intel.com/834254/mlc_v3.11b.tgz",
+        sha256="5d5abd27d145af59d564c9c53938301a08205f075b9bd41f2f92b4c8efeb2824",
+        preferred=True,
+    )
+
+    version(
+        "3.11a",
+        url="https://downloadmirror.intel.com/822971/mlc_v3.11a.tgz",
+        sha256="e5fe60c370962a22028e3da4d6e093fde232f92f06aa217e0eac602533d94ef1",
+    )
+
+    version(
+        "3.11",
+        url="https://downloadmirror.intel.com/793041/mlc_v3.11.tgz",
+        sha256="e34d246a52a1d2d4461dc156e073b81fa142d30fde172ca412eb77ad73376cf5",
+    )
+
+    version(
+        "3.10",
+        url="https://downloadmirror.intel.com/763324/mlc_v3.10.tgz",
+        sha256="39bdc467d134ed43561e622efe664d62cdee147bf87bf3090f2a56f86313d789",
+    )
+
+    version(
+        "3.9a",
+        url="https://downloadmirror.intel.com/736634/mlc_v3.9a.tgz",
+        sha256="1036b9579eaa08f431802bb879d7588960eb93bbe4612d238ac9c6d34e0bfa34",
+    )
```

Comment:
I cannot confirm the sha256s for the last two versions. If I try to access the URLs directly I get `Access denied` errors. If I try to stage them, the concretizer fails (even when I use `+license-agreed`).

Reply:
Thanks, they must have removed them since I made this PR originally.

I removed the two oldest ones since I get the same error you do trying to access them.